### PR TITLE
Added sister projects

### LINKED ISSUE #1654: Add more sister projects
**Actual Behaviour**

not all prjects in menu bar

**Expected Behaviour**

More were added

**Steps to reproduce it**

Update html file add image

**Would you like to work on the issue?**

Yes I am

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes #1654 
Changes: Updated index.html and added icons

### CODE REVIEW

**Comment by geekyd** [comment]
@arjunpandey2003 Please share screen shots

**Comment by arjunpandey2003** [comment]
yes deploying with heroku

**Comment by arjunpandey2003** [comment]
https://salty-retreat-75405.herokuapp.com/

**Comment by arjunpandey2003** [comment]
@geekyd sent please review and merge :)

File: src/www/index.html [hunk 1]
```diff
@@ -202,6 +202,14 @@
           <a class="custom-icon" href="https://loklak.org/" target="_blank"><img src="./icons/loklak.png">
           <p class="custom-title">loklak</p></a>
         </div>
+        <div class="custom-menu-item">
+          <a class="custom-icon" href="https://phimp.me/" target="_blank"><img src="./icons/phimp.png">
+          <p class="custom-title">Phimp.me</p></a>
+        </div>
+        <div class="custom-menu-item">
```
Comment (geekyd) [inline src/www/index.html]: @arjunpandey2003 Please add a square logo, this logo does not scale properly.

Comment (arjunpandey2003) [reply src/www/index.html]: sure

Comment (arjunpandey2003) [reply src/www/index.html]: should i increase the width

**Comment by arjunpandey2003** [comment]
pls check my dimensions

**Comment by arjunpandey2003** [comment]
should i deploy again

**Comment by arjunpandey2003** [comment]
@geekyd

File: src/www/index.html (hunk 1 above)
Comment (geekyd) [reply src/www/index.html]: @arjunpandey2003 Please use a square logo. Do not change width

**Comment by arjunpandey2003** [comment]
@geekyd done giving link

**Comment by arjunpandey2003** [comment]
sorry added the wrong image

**Comment by arjunpandey2003** [comment]
Hey I cant squash commits as my laptop has gone for repair and dont have enough net to download github desktop pls understand my situation. Thanks can you squash and merge

**Comment by arjunpandey2003** [comment]
https://still-meadow-62504.herokuapp.com

**Comment by geekyd** [comment]
@arjunpandey2003 Nice work. Please check the screenshot. The logos are not square which does not scale well. Use 60 x 60 size 
screen shot 2018-01-14 at 4 03 01 pm

**Comment by arjunpandey2003** [comment]
thanks @geekyd but isnt 60x60 too small.

**Comment by arjunpandey2003** [comment]
thanks pls merge after tests. first pr!!!

**Pull request merged**